test/TwabController.t.sol

```diff
@@ -300,6 +300,64 @@ contract TwabControllerTest is BaseTest {
     );
   }
 
+  function testGetTimestampPeriod() public {
+    assertEq(twabController.getTimestampPeriod(PERIOD_OFFSET), 0);
+    assertEq(twabController.getTimestampPeriod(PERIOD_OFFSET-1), 0);
+    assertEq(twabController.getTimestampPeriod(PERIOD_OFFSET+PERIOD_LENGTH), 1);
+  }
+
+  function testCurrentOverwritePeriodStartedAt_beginning() public {
+    assertEq(twabController.currentOverwritePeriodStartedAt(), PERIOD_OFFSET);
+  }
+
+  function testCurrentOverwritePeriodStartedAt_partway() public {
+    vm.warp(PERIOD_OFFSET + PERIOD_LENGTH/2);
+    assertEq(twabController.currentOverwritePeriodStartedAt(), PERIOD_OFFSET);
+  }
+
+  function testCurrentOverwritePeriodStartedAt_next() public {
+    vm.warp(PERIOD_OFFSET + PERIOD_LENGTH);
+    assertEq(twabController.currentOverwritePeriodStartedAt(), PERIOD_OFFSET + PERIOD_LENGTH);
+  }
+
+  /// It should not be possible for a twab measurement to change after the period has finalized.
+  function testGetTotalSupplyTwabBetween_regression() public {
+    uint32 secondPeriodStart = PERIOD_OFFSET + PERIOD_LENGTH;
+    vm.warp(secondPeriodStart);
+
+    twabController.mint(alice, 1000e18);
+
+    vm.warp(secondPeriodStart + PERIOD_LENGTH);
+
+    twabController.mint(alice, 1000e18);
+
+    vm.warp(secondPeriodStart + PERIOD_LENGTH * 2);
+
+    uint256 balance = twabController.getTotalSupplyTwabBetween(
+      address(this),
+      secondPeriodStart + PERIOD_LENGTH,
+      secondPeriodStart + PERIOD_LENGTH*2
+    );
+
+    assertGt(balance, 0);
```

Comment:
What is the total supply balance here? `2000e18` ?

Reply:
Yes; eq 2000e18